quickwit-cli/src/cli.yaml

```diff
@@ -12,17 +12,24 @@ subcommands:
                 about: Creates an index.
                 display_order: 1
                 args:
-                    - metastore-uri:
-                        about: Location of the metastore.
-                        long: metastore-uri
-                        value_name: METASTORE URI
+                    - index-id:
+                        about: ID of the target index.
+                        long: index-id
+                        value_name: INDEX ID
                         required: true
-                        env: QUICKWIT_METASTORE_URI
                     - index-config-uri:
```

Comment:
It is true that we accept an uri here. But considering the main usage is just to pass a file from the local file system, 
should we call the CLI option simply `index-config` ?

Reply:
(We could even make that a positional argument.)

Reply:
`index-config` is good.